app/rdf/query-search.ts

```diff
@@ -342,4 +436,5 @@ const mkScoresQuery = (
       }
   `.GROUP().BY`?iri`.THEN.BY`?title`.THEN.BY`?status`.THEN.BY`?datePublished`
     .THEN.BY`?description`.THEN.BY`?publisher`.THEN.BY`?creatorIri`.THEN
-    .BY`?creatorLabel`.prologue`${pragmas}`.prologue`#pragma join.bind off`; // HOTFIX WRT Stardog v9.2.1 bug see https://control.vshn.net/tickets/sbar-1066
+    .BY`?creatorLabel`.prologue`${pragmas}`.prologue`#pragma join.bind off`;
+}; // HOTFIX WRT Stardog v9.2.1 bug see https://control.vshn.net/tickets/sbar-1066
```

Comment:
Nit: maybe it's possible to reformat the chaining above to apply this comment just above the prologue`#pragma join.bind off`, as it's connected to this (now it looks like it applies to the whole function, as it's outside of it)?